_docs/resources/index.md

```diff
@@ -1,7 +1,111 @@
+    <a id="link" href="{{site.baseurl}}/resources/sketch-library.html">Sketch Library set up instructions</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div ><img src="{{site.baseurl}}/assets/images/lo-fi.svg" alt="designer UX icon"/>
+  	<h4 style="margin-top: 16px; margin-bottom: 16px;">Low-fi Stickersheet</h4>
+  </div>
+    <p style="margin-bottom: 16px">The Low-fi stickersheet is a downloadable sketch file. Once downloaded a designer can copy and paste elements from this file into your own Sketch documents to quickly prototype designs for stakeholders and designers. </p>
+    <a id="link" href="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/sketch/low-fi_helix_stickersheet_v0.1.sketch">Download Low-fi Sticker Sheet</a>
+</div>
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Online Tools
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/palette-contrast.svg" alt="designer UX icon"/>
+    <h4 style="margin-top: 16px; margin-bottom: 16px;">Palette contrast checker</h4>
+  </div>
+    <p style="margin-bottom: 16px">When finalizing color schemes for a component or layout accessibility is always a concern. This light-weight browser app lets you select a color for the Helix palette and check readability against different backgrounds.</p>
+    <a id="link" target="_blank" href="http://citguy.com/PaletteContrast">View the tool</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/codepen.svg" alt="designer UX icon"/>
+    <h4 style="margin-top: 16px; margin-bottom: 16px;">Codepen Template</h4>
+  </div>
+    <p style="margin-bottom: 16px">If you are looking to test out Helix-UI web-components look no further than this pen. The Helix team has created a codepen template that requires all the necesary polyfills and Helix-UI code for quick and easy testing.</p>
+    <a id="link" target="_blank" href="https://codepen.io/pen?template=YaRqQg">Create a pen</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="designer UX icon"/>
+    <h4 style="margin-top: 16px; margin-bottom: 16px;">Design system repo</h4>
+  </div>
+    <p style="margin-bottom: 16px">The repository used to generate the Helix documentation site (this site). Feel free to star, submit issues, or help. Note: You will need access to the rackerlabs organization on public git to view.</p>
```

Comment:
s/`The repository used to`/`This repository is used`

Reply:
updated.